docs/starters.yml

```diff
@@ -3624,3 +3624,16 @@
     - Includes Search Feature.
     - Syntax Highlight in Code.
     - Styling in Bootstrap
+- url: https://gastby-starter-organization.netlify.com/
```

Comment:
Typo in url 😆

Reply:
oh my, why did I not use copy paste 😂, i'll update

Reply:
Haha too late! It's in now 

The url works so don't worry about it 😉